launch_ros/launch_ros/actions/composable_node_container.py

```diff
@@ -51,6 +56,26 @@ def __init__(
         super().__init__(name=name, namespace=namespace, **kwargs)
         self.__composable_node_descriptions = composable_node_descriptions
 
+    @classmethod
+    def parse(cls, entity: Entity, parser: Parser):
+        """Parse node_container."""
+        _, kwargs = super().parse(entity, parser)
+
+        kwargs['package'] = parser.parse_substitution(entity.get_attr('pkg', data_type=str))
+        kwargs['executable'] = parser.parse_substitution(entity.get_attr('exec', data_type=str))
+        kwargs['name'] = parser.parse_substitution(entity.get_attr('name', data_type=str))
+        kwargs['namespace'] = parser.parse_substitution(
+            entity.get_attr('namespace', data_type=str))
```

Comment:
@kenji-miyake if I'm not mistaken, these attributes are already parsed by the `Node` parsing method. Why duplicate?

Reply:
hmm, I see. I'll try to remove them.

Reply:
You are correct. Removed in b92e4c4.